src/shared/components/ui/select/SelectContent.tsx

```diff
@@ -0,0 +1,53 @@
+import { ReactNode } from 'react';
+
+import { cn } from '../../../libs/cn';
+import { useSelect } from './useSelect';
+
+/**
+ * 드롭다운 컨텐츠 컨테이너 컴포넌트
+ * SelectItem들을 포함하며, 백드롭 클릭 시 드롭다운이 닫힙니다.
+ *
+ * @param children - 드롭다운 내부에 표시될 SelectItem들
+ * @param className - 추가 CSS 클래스명
+ *
+ * @example
+ * ```tsx
+ * <SelectContent>
+ *   <SelectItem value="option1">옵션 1</SelectItem>
+ *   <SelectItem value="option2">옵션 2</SelectItem>
+ * </SelectContent>
+ * ```
+ */
+export default function SelectContent({
+  children,
+  className,
+}: {
+  children: ReactNode;
+  className?: string;
+}) {
+  const { isOpen, setIsOpen } = useSelect();
+
+  return (
+    <>
+      {/* 백드롭 */}
+      {isOpen && (
+        <div
+          className='fixed inset-0 z-90'
+          onClick={() => setIsOpen(false)}
+          role='presentation'
+        />
+      )}
+      {/* 드롭다운 */}
+      {isOpen && (
+        <div
+          className={cn(
+            'absolute top-full right-0 left-0 z-100 mt-1 max-h-[30rem] overflow-y-auto rounded border bg-white shadow-lg transition-all duration-300',
```

Comment:
UI 상 `border-radius`가 더 들어가면 어떨지 제안해봅니다! `SelectBox`와 동일한 `border-radus`가 들어가면 UI가 더 예뻐질 거 같아요.

그리고 지금 `border`만 적용되어 있는데 이 부분도 figma 디자인 보면 `gray-100`으로 적용되어 있으니 맞춰주시면 좋을 거 같습니다!
```suggestion
            'absolute top-full right-0 left-0 z-100 mt-1 max-h-[30rem] overflow-y-auto rounded-2xl border border-gray-100 bg-white shadow-lg transition-all duration-300',
```